content/en/docs/reference/glossary/cloud-provider.md

```diff
@@ -1,18 +1,32 @@
 ---
-title: Cloud Provider
+title: Cloud provider
 id: cloud-provider
 date: 2018-04-12
 full_link: /docs/concepts/cluster-administration/cloud-providers
 short_description: >
-  Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+  A company that offers a cloud computing platform
 
-aka: 
+aka:
+- Cloud service provider
 tags:
 - community
 ---
- Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+ A company that offers a cloud computing platform.
 
-<!--more--> 
+<!--more-->
 
-Cloud providers or sometime called Cloud Service Provider (CSPs) provides cloud computing platforms.  They may offer services such as Infrastructure as a Service (IaaS) or Platform as a Service (PaaS).  Cloud providers host the Kubernetes cluster and also provide services that interact with the cluster, such as Load Balancers, Storage Classes etc. 
+Cloud providers, sometime called Cloud Service Providers (CSPs), offer
+cloud computing platforms or services.
 
+Your cloud provider might offer managed infrastructure (also called
+Infrastructure as a Service or IaaS).
```

Comment:
Line 21: Possible edit: s/Your cloud provider might offer/Cloud providers offer managed infrastructure (also ..)/
Line 23: Add comma after storage? With managed infrastructure, .. storage, and networking
Line 27: s/manager/managed/
Line 28: capitalize nodes or not?
Line 31: Could move/delete this line and combined with Line 27:
You can also find Kubernetes as a managed service (also known as Platform as a Service, or PaaS).

Reply:
Thanks for the feedback. All sorted I think / hope.